autoload/dirvish.vim

```diff
@@ -388,6 +388,7 @@ function! dirvish#open(...) range abort
     return
   endif
   if !&hidden && &modified
+      \ && (!exists("*win_findbuf") || len(win_findbuf(winbufnr(0))) == 1)
```

Comment:
Thanks. The usual way to get the current buffer number is `bufnr('%')`. Was `winbufnr(0)` chosen intentionally for some reason?